tasks/main.yml

```diff
@@ -10,6 +10,28 @@
     - nginx-revproxy
     - packages
 
+- name: Install python-passlib
+  apt:
+    name:
+        - "python-passlib"
```

Comment:
Why do we need this packages?
As in the ansible docs, should't it need on the target host?

    The below requirements are needed on the host that executes this module.
    passlib>=1.6